src/networks/mod.rs

```diff
@@ -263,11 +266,11 @@ impl ChainConfig {
             f3_consensus: false,
             f3_bootstrap_epoch: -1,
             f3_initial_power_table: Default::default(),
-            f3_manifest_server: Some(
-                "12D3KooWENMwUF9YxvQxar7uBWJtZkA6amvK4xWmKXfSiHUo2Qq7"
-                    .parse()
-                    .expect("Invalid PeerId"),
+            f3_contract_address: Some(
+                EthAddress::from_str("0x476AC9256b9921C9C6a0fC237B7fE05fe9874F50")
```

Comment:
The poll interval is managed via an environment variable. Would it make sense to expose the same functionality for the contract address?

Reply:
Fixed.